### ISSUE TITLE
It possible to unlock hidden dlc for some game?

### ISSUE THREAD

**Issue by nos4r3** [st2024/Steamtools#11]
Hi, My Time at Sandrock have some hidden dlc that you can't find it in steamdb dlc page, it contain kickstarter content like fox and lots of exclusive stuff so even if i drag all dlc in the steamdb from my time at sandrock to steamtools it will not unlock those hidden dlc.

While if you using koalageddon you can unlock all of them because koala always attempt to unlock every dlc in the game, but i don't like using kolageddon because some game detect it and i have to uninstalled to play that game again and it feel tedious, i prefer steamtool more, so anyway to make steamtool unlock every dlc the game have? (creamapi also the same that if you set to unlockall it will unlock these hidden dlc too)

Also how to get sponsor version that can download missing dlc too?

**Comment by st2024** [comment]
Yes, the theory is feasible, but you need to know the app IDs of those DLCs.
